### DISCUSSION TITLE
Get some inspiration from this related project

### DISCUSSION THREAD

**Discussion by RoneoOrg** [hugomods/icons#70] [Ideas]
Hi there,

and thanks for sharing this module.
I've been working on a related project, and you may find some interesting bits, like unamed parameters, custom CSS classes or injection of `aria-hidden="true"` in each loaded SVG

See [this post](https://roneo.org/en/hugo-svg-icon-shortcode/) for details, cheers!

**Comment by razonyang** [comment]
Yep, the module supports the features you mentioned, it would be great to add the `aria-hidden` attribute on inline SVG (done by #69).
Thanks for sharing your ideas.
